docs/user-guide.md

```diff
@@ -0,0 +1,72 @@
+# User Guide
+
+## Features
+
+### How to verify a kani harness using the extension
+
+
+As soon as the rust file containing the proofs is opened using the Kani extension in a VSCode instance, you should see the Kani proofs loaded as regular unit tests in the Testing Panel on the left border of VSCode. This is how the testing page looks like when you click on the panel.
+
+![Verify Proofs](../resources/screenshots/first.png)
+
+### Running Proofs
```

Comment:
This entire section looks like a workflow, actually. What is it that you want to describe, features or workflows?